ckcp/openshift_dev_setup.sh

```diff
@@ -191,32 +191,32 @@ install_openshift_gitops() {
     ARGOCD_HOSTNAME="$(${test_cmd})"
   done
   echo "OK"
-  echo "  - Argo CD URL: https://$ARGOCD_HOSTNAME"
+  echo "- Argo CD URL: https://$ARGOCD_HOSTNAME"
 
   #############################################################################
   # Post install
   #############################################################################
   # Log into Argo CD
-  echo -n "  - Argo CD Login: "
+  echo -n "- Argo CD Login: "
   local argocd_password
   argocd_password="$(kubectl get secret openshift-gitops-cluster -n $ns -o jsonpath="{.data.admin\.password}" | base64 --decode)"
   argocd login "$ARGOCD_HOSTNAME" --grpc-web --insecure --username admin --password "$argocd_password" >/dev/null
   echo "OK"
 
   # Register the host cluster as pipeline-cluster
   local cluster_name="plnsvc"
-  echo "  - Register host cluster to ArgoCD as '$cluster_name': "
+  echo -n "- Register host cluster to ArgoCD as '$cluster_name': "
```

Comment:
I think the previous version will look nicer if the cluster is registered. Or you could change it to:
```
echo -n "..."
if ! KUBECONFIG ... ; then
  echo
  argocd ...
else
  echo "OK"
fi
```

Reply:
If the cluster is already registered it will look like:
previous version:
```
  - Register host cluster to ArgoCD as 'plnsvc': 
    OK
```

This version

```
  - Register host cluster to ArgoCD as 'plnsvc': OK
```

Reply:
Sorry, I meant NOT registered. My understanding is that the new version might look like:
```
  - Register host cluster to ArgoCD as 'plnsvc': INFO[0000] ServiceAccount "argocd-manager" created in namespace "kube-system" 
INFO[0000] ClusterRole "argocd-manager-role" created    
INFO[0000] ClusterRoleBinding "argocd-manager-role-binding" created 
OK
```
and I think it would look better as:
```
  - Register host cluster to ArgoCD as 'plnsvc': 
INFO[0000] ServiceAccount "argocd-manager" created in namespace "kube-system" 
INFO[0000] ClusterRole "argocd-manager-role" created    
INFO[0000] ClusterRoleBinding "argocd-manager-role-binding" created 
```

Reply:
Got it, I have conditioned the output to handle it.